robot/Cumulus/resources/ObjectMangerPageObject.py

```diff
@@ -149,4 +149,44 @@ def create_custom_field(self, **kwargs):
 			elif type.lower() == 'formula':
 				self.create_formula_field(kwargs['Field_Name'],kwargs['Formula'])
 			elif type.lower() == 'text':
-				self.create_text_field(kwargs['Field_Name'])
\ No newline at end of file
+				self.create_text_field(kwargs['Field_Name'])
+		
+	
+	@capture_screenshot_on_error
+	def add_picklist_values(self, **kwargs):
```

Comment:
what would happen if the same keyword is executed again and again in the same org?

Reply:
It will fail may be because there would be a duplicate constraint. But on metaCI the org is in a clean state so there won't be any problem. The testcase specially wants these values and that is part of the acceptance criteria.

Reply:
I was not sure if there is a way to delete these picklist values from UI

Reply:
Modified it to have unique picklist values so that rerunning test works always